entries/d/dotdigital.com.json

```
@@ -0,0 +1,12 @@
+{
+  "dotdigital": {
+    "domain": "dotdigital.com",
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://support.dotdigital.com/hc/en-gb/articles/212212678-Secure-your-account-with-two-factor-authentication",

```

For Zendesk URLs the part after the "-" is redundant and only causes issues when someone decides to rename the page.
```suggestion
    "documentation": "https://support.dotdigital.com/hc/en-gb/articles/212212678",

```

---

That's good to know on ZendDesk URLs. I'll make a note of that. I'll amend this one.